plugins/nvm/nvm.plugin.zsh

```diff
@@ -1,3 +1,4 @@
 # The addition 'nvm install' attempts in ~/.profile
 
 [[ -s ~/.nvm/nvm.sh ]] && . ~/.nvm/nvm.sh
+[[ -r $NVM_DIR/bash_completion ]] && . $NVM_DIR/bash_completion
```

Comment:
Does this work out of the box? To load bash completion files, [`bashcompinit`](http://zsh.sourceforge.net/Doc/Release/Completion-System.html#index-bashcompinit) needs to be called.